### PR TITLE
user feedback if template missing

### PR DESCRIPTION
**Pull request by Hornebom**
### Description
- display information to user if no asset template is found
- provide action button to user

Issue: https://github.com/centrifuge/apps/issues/1301

### Approvals
- [ ] Dev
- [ ] Product

### CODE REVIEW

File: centrifuge-app/src/pages/IssuerPool/Assets/CreateLoan.tsx [hunk 1]
```diff
@@ -338,16 +339,34 @@ function IssuerCreateLoan() {
...
+                {!templateId && (
+                  <Box
+                    mb={3}
+                    py={2}
+                    borderWidth={0}
+                    borderBottomWidth={1}
+                    borderColor="borderPrimary"
+                    borderStyle="solid"
+                  >
+                    <Text>Asset template is missing. Please add an asset template or create a new one.</Text>
```
Comment (onnovisser) [inline centrifuge-app/src/pages/IssuerPool/Assets/CreateLoan.tsx:352]: The copy on this seems off to me. the only way to "add" a template is to create a new one. I would rephrase it to
`Asset template is missing. Please create one first.`

**Review by sophialittlejohn — APPROVED** [review]
